desktop/linux/install/fedora.md

```diff
@@ -1,33 +1,25 @@
 ---
 description: Instructions for installing Docker Desktop on Fedora
-keywords: requirements, apt, installation, fedora, rpm, install, uninstall, upgrade, update
+keywords: fedora, rpm, update install, uninstall, upgrade, update, linux, desktop, docker desktop, docker desktop for linux, dd4l
 title: Install Docker Desktop on Fedora
 toc_max: 4
 ---
 
-To get started with Docker Desktop on Fedora, make sure you
-[meet the prerequisites](#prerequisites), then
-[install Docker Desktop](#install-docker-desktop).
+This page contains information on how to install, launch and upgrade Docker Desktop on a Fedora distribution.
 
 ## Prerequisites
 
-Your Fedora distribution must meet the [system requirements](../install.md#system-requirements) to install and launch Docker Desktop successfully.
+To install Docker Desktop successfully, you must:
+- Meet the [system requirements](../install.md#system-requirements).
+- Have a 64-bit version of either Fedora 35 or Fedora 36.
 
-Additionally, for a Gnome Desktop environment you must install AppIndicator and KStatusNotifierItem [Gnome extensions](https://extensions.gnome.org/extension/615/appindicator-support/).
-
-
-### OS requirements
-
-To install Docker Desktop for Linux, you need the 64-bit version of one of these Fedora versions:
-
-- Fedora 35
-- Fedora 36
+For a Gnome Desktop environment you must also install AppIndicator and KStatusNotifierItem [Gnome extensions](https://extensions.gnome.org/extension/615/appindicator-support/).
 
 ## Install Docker Desktop
 
-Recommended approach to install Docker Desktop on Ubuntu:
+To install Docker Desktop on Fedora:
 
-1. [Set up Docker's RPM repository](../../../engine/install/fedora.md#set-up-the-repository) 
+1. Set up [Docker's RPM repository](../../../engine/install/fedora.md#set-up-the-repository). 
```

Comment:
Wondering if we should use "package repository" for these (instead of `RPM` / `DEB`) (not sure if those should be all-caps  🤔and using "package repository" avoids that 😂